### DISCUSSION TITLE
Celery with Support for GCP Pub/Sub

### DISCUSSION THREAD

**Discussion by sb-ghvcs** [celery/kombu#2155] [Q&A] [answered]
Hi!
I am looking to get started with celery, but since my usecase is pretty minimal, I don't want the hassle of setting up a MQ/Redis server. I already have prior experience with GCP and was looking into celery with pubsub as a broker. I found out that there is a recently merged PR (https://github.com/celery/kombu/pull/2147) adding this support.

Is there an ETA on the release of the new version with this support? If it's within the week I'll hold off on getting started.
Else, could someone point me to building it myself? I see the last release of celery happened in April. There is a v5.5.0rc1 pre-release of [kombu](https://github.com/celery/kombu/releases/tag/v5.5.0rc1) and of [celery](https://github.com/celery/celery/releases/tag/v5.5.0rc1), but the celery pre-release is a few days older than kombu and doesn't mention pubsub in the changelog.

Thanks!

**Comment by Nusnus** [comment, marked as answer]
1. Celery is waiting for this to be merged (today): https://github.com/celery/celery/pull/9351
2. Kombu will be officially released within the next two weeks (even less, probably much less).
3. Celery will be released by the end of October and will include Kombu 5.5 final release.

**Reply by sb-ghvcs** [reply]
Perfect that works. Thanks a ton!
